test-data/local/configuration_tests/ensemble_config.ert

```diff
@@ -0,0 +1,27 @@
+JOBNAME  Job%d
+RUNPATH /tmp/simulations/run%d
+NUM_REALIZATIONS 10
+MIN_REALIZATIONS 10
+
+GEN_PARAM         GP              GP.txt  INIT_FILES:GP/GP.txt  INPUT_FORMAT:ASCII   OUTPUT_FORMAT:ASCII
+
+CUSTOM_KW SNAKE_OIL_NPV snake_oil_npv.txt
+
+GEN_DATA SNAKE_OIL_OPR_DIFF INPUT_FORMAT:ASCII RESULT_FILE:snake_oil_opr_diff_%d.txt REPORT_STEPS:199
+
+GEN_DATA SNAKE_OIL_GPR_DIFF INPUT_FORMAT:ASCII RESULT_FILE:snake_oil_gpr_diff_%d.txt REPORT_STEPS:199
+
+-- GEN_KW MULTFLT configuration_tests/FAULT_TEMPLATE configuration_tests/MULTFLT.INC configuration_tests/MULTFLT.TXT
+GEN_KW MULTFLT FAULT_TEMPLATE MULTFLT.INC MULTFLT.TXT
+
+SURFACE TOP INIT_FILES:configuration_tests/surface/small.irap OUTPUT_FILE:configuration_tests/surface/small_out.irap BASE_SURFACE:configuration_tests/surface/small.irap
+
+GRID grid/CASE.EGRID
+
+FIELD PERMX PARAMETER  permx.grdcel  INIT_FILES:fields/permx%d.grdecl
+
+-- SCHEDULE_PREDICTION_FILE input/schedule.sch INIT_FILES:input/test_%d.sh
```

Comment:
rmcl

Reply:
It remains I just up-worded the comment :)